RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHGC_cfi.py

```diff
@@ -48,7 +48,11 @@
     algoName = cms.string("PFClusterFromHGCalMultiCluster"),
     thresholdsByDetector = cms.VPSet(
     ),
-    clusterSrc = cms.InputTag("hgcalMultiClusters")
```

Comment:
Does this change imply that the "old" HGCal clustering steps (`hgcalLayerClusters` and `hgcalMultiClusters`) are no longer used at all and could be removed from standard workflows? @felicepantaleo @rovere

Reply:
Ciao @kpedro88 after this PR is merged we will start the process of trimming out from the release all the bits and pieces that are no longer necessary.